### PR TITLE
fix(NSWG-ECO-428): update current status

### PR DESCRIPTION
**Pull request by ChALkeR**
/cc @MylesBorins

### CODE REVIEW

File: vuln/npm/428.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
   "module_name": "base64url",
   "cves": [],
   "vulnerable_versions": "<=2.0.0",
-  "patched_versions": null,
-  "recommendation": "No fix is currently available for this vulnerability.\n\nIt is our recommendation to not install or use this module at this time.",
-  "references": "- https://hackerone.com/reports/321687",
+  "patched_versions": ">=3.0.0",
+  "recommendation": "Update `base64url` module to 3.0.0 or higher.",
```
Comment (lirantal) [inline vuln/npm/428.json]: We're generally not wrapping the module name so just a plain text would be fine here

**Comment by ChALkeR** [comment]
@lirantal 
> We're generally not wrapping the module name so just a plain text would be fine here

I fixed that, though look at the `overview` for this entry — it uses backticks to wrap the module name.

**Comment by lirantal** [comment]
Right. In https://github.com/nodejs/security-wg/issues/200 we agreed on the `overview` field not being a strict plain-text.

**Comment by ChALkeR** [comment]
But isn't recommendarion also markdown?

**Pull request merged**